cypress/e2e/spec.cy.js

```diff
@@ -1,6 +1,20 @@
-describe('template spec', () => {
-  it('passes', () => {
+describe('opening a checkout', () => {
+  it('can open a checkout for the Demo store', () => {
     cy.visit('http://127.0.0.1:8080/dist/index')
     cy.window().should('have.attr', 'PurpleDot');
+
+    cy.window().then((win) => {
+      win.PurpleDot.init({ apiKey: 'b351faa2-8693-4c09-b814-759beed90d0b' });
+
+      win.PurpleDot.checkout.open({
+        cartId: 'bcc9daa54d4eb89b36df5321dd087ab2',
+      })
```

Comment:
Could we read this `cartId` from the cookie or from the API ourselves? If the integration is using the Shopify cart, they probably won't know where to get this, whereas we can get it quite easily. Then it just becomes `PurpleDot.checkout.open()`.

Reply:
Reading the rest of the code, should we use `getShopifyAJAXCartID()` here?